src/crud.js

```diff
@@ -33,32 +33,35 @@ models: {
 
 export default (server, model, { prefix, defaultConfig: config, models: permissions }) => {
   const modelName = model._singular;
+  const permissionsIsArray = Array.isArray(permissions);
```

Comment:
If permissions must be an array, why should we check it in all our conditions? Maybe we should bail out if permissions is neither undefined nor an array?